package.json

```diff
@@ -5,29 +5,27 @@
   "repository": "https://github.com/DimensionDev/Stego-JS",
   "license": "MIT",
   "author": "DimensionDev",
-  "main": "es/node.js",
-  "browser": "es/dom.js",
-  "module": "es/node.js",
-  "types": "es/node.d.ts",
+  "main": "./cjs/node.js",
+  "module": "./esm/node.js",
+  "browser": "./cjs/dom.js",
   "bin": "./bin/stego.js",
-  "files": [
-    "bin",
-    "es",
-    "umd"
-  ],
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm run build:umd && npm run build:es",
-    "build:es": "tsc -d",
+    "build": "npm run build:umd && npm run build:dist",
+    "build:dist": "tsc && tsc -p tsconfig.esm.json",
```

Comment:
```suggestion
    "build": "npm run build:umd && npm run build:esm",
    "build:esm": "tsc && tsc -p tsconfig.esm.json",
```

Reply:
The `build:dist` include `commonjs` and `esmodule` output